src/Nri/Ui/Menu/V4.elm

```diff
@@ -718,13 +748,15 @@ viewEntry config focusAndToggle { upId, downId, entry_ } =
             entryContainer
                 [ class "MenuEntryContainer"
                 , css
-                    [ padding2 (px 5) zero
-                    , position relative
-                    , firstChild
+                    ([ padding2 (px 5) zero
+                     , position relative
+                     , firstChild
                         [ paddingTop zero ]
-                    , lastChild
+                     , lastChild
                         [ paddingBottom zero ]
-                    ]
+                     ]
+                        ++ config.entryContainerCss
```

Comment:
I'm surprised elm-review didn't complain about this `++` and the one in `styleGroupContainer` we are explicitly using `::` in `styleContainer`.

Reply:
Oh I see what you mean yeah. This one does not mind I guess because there was already the longer list of styles here. I think the elm review rule you speak of fires for `++` with a list of a known single element. As to why I changed `styleContainer` the way I did... I honestly don't remember why I did that one a different way lol. I think my brain saw it as oh theres only a couple elements to cons onto the incomming list here so I'll just do that versus the other side already having decent amount in the list 🤷